app/helpers/application_helper.rb

```diff
@@ -76,6 +76,54 @@ def format_datetime(date, format = :long)
   def time_period_class(item)
     return 'not-yet-valid' if item.not_yet_valid?
     return 'currently-valid' if item.currently_valid?
-    return 'expired' if item.expired?
+    'expired' if item.expired?
+  end
+
+  # Sets the title for a page.
+  #
+  # @param [String] text The title to set.
+  # @return [String]
+  def title(text)
+    content_for(:page_title, text)
+  end
+
+  # Sets the title of a page to the +.header+ key.
+  #
+  # @return [String]
+  def default_title
+    title(t('.header'))
```

Comment:
Actually this should be built on the breadcrumbs, and reversed. See CONTRIBUTING.md